guides/common/modules/proc_creating-a-content-view-short.adoc

```diff
@@ -4,7 +4,8 @@
 Use the following procedure to create a Content View.
 
 .Procedure
-. In the {ProjectWebUI}, navigate to *Content* > *Content Views* and click *Create New View*.
+. In the {ProjectWebUI}, navigate to *Content* > *Content Views* and click *Create Content View*.
 . In the *Name* field, enter *CentOS Content View*.
 . In the *Description* field, enter a description for the Content View.
-. Click *Save*.
+. In the *Type* field, choose a *Component content view* or a *Composite content view*.
```

Comment:
```suggestion
. In the *Type* field, select a Component Content View or a Composite Content View.  
```
You only highlight actual graphical interface elements.

It's never choose, always select. As per IBM style. Use the graphical interface elements chapter.